vowpalwabbit/accumulate.cc

```diff
@@ -118,11 +118,6 @@ void do_weighting(vw& all, uint64_t length, float* local_weights, T& weights)
 
 void accumulate_weighted_avg(vw& all, parameters& weights)
 {
```

Comment:
Removing a warning seems a bit worrisome.  Can we stick the information for the weight layout into 'parameters' and then reinstate the check?